encryption/x25519/scalar.go

```diff
@@ -3,8 +3,10 @@ package x25519
 import (
 	"bytes"
 	"crypto/sha512"
-	"github.com/ElrondNetwork/elrond-go-core/core/check"
-	crypto "github.com/ElrondNetwork/elrond-go-crypto"
+
+	"github.com/multiversx/mx-chain-core-go/core/check"
+	crypto "github.com/multiversx/mx-chain-crypto-go"
+
```

Comment:
remove all spaces and sort imports again - I think it's not completely done

Reply:
pushed